src/Microsoft.Android.Sdk.ILLink/SetupStep.cs

```diff
@@ -43,9 +46,13 @@ protected override void Process ()
 			subSteps2.Add (new PreserveRegistrations (cache));
 			subSteps2.Add (new PreserveJavaInterfaces ());
 
-			InsertAfter (new FixAbstractMethodsStep (cache), "RemoveUnreachableBlocksStep");
-			InsertAfter (subSteps2, "RemoveUnreachableBlocksStep");
-			InsertAfter (subSteps1, "RemoveUnreachableBlocksStep");
+			InsertAfter (new FixAbstractMethodsStep (cache), "SetupStep");
+			InsertAfter (subSteps2, "SetupStep");
+			InsertAfter (subSteps1, "SetupStep");
+
+			// temporary workaround: this call forces illink to process all the assemblies
+			foreach (var assembly in (IEnumerable<AssemblyDefinition>)getReferencedAssembliesMethod.Invoke (Context, null))
```

Comment:
Should we have a `null` check for  `getReferencedAssembliesMethod` and throw, just so if the method disappears we'll get a better error?

Reply:
Should not harm, even if it is short lived temporary.